app/src/main/java/io/intrepid/skeleton/utils/RxUtils.java

```diff
@@ -0,0 +1,25 @@
+package io.intrepid.skeleton.utils;
+
+import android.support.annotation.NonNull;
+
+import java.util.List;
+
+import rx.functions.Action1;
+import rx.functions.Func1;
+import timber.log.Timber;
+
+public class RxUtils {
+
```

Comment:
Yeah we can add them in a separate PR. (although don't go crazy here, otherwise we might need to extract it out to a library)